public/js/admin/registers.js

```diff
@@ -17,8 +33,16 @@ const {
     '.toEmails',
     '.main--form',
     '.from-msg',
+    '.profile',
+    '.profile-a',
+    '.registers',
+    '.registers-a',
   ],
 );
+profile.classList.remove('focus');
+profileA.classList.remove('black');
```

Comment:
@aminalakhsham You could have the black styling on tab `a` elements come from the `focus` class instead of needing an extra `black` class:
```
.tab.focus a {
 //add the black styles here
}
```

And then you wouldn't need to add or remove two classes every time, you could just add or remove the 'focus' class to the tab.